src/utils/test/token_bucket_throttling_controller_test.cpp

```diff
@@ -109,7 +110,8 @@ class token_bucket_throttling_controller_test : public ::testing::Test
         for (int i = 0; i < 100000; i++) {
             token_bucket->consumeWithBorrowAndWait(
                 1, throttle_limit / partition_count * 0.8, throttle_limit / partition_count * 1.0);
-            if (!cntl->get_token(1)) {
+            cntl->get_token(1);
```

Comment:
why did you add `cntl->get_token(1)` here?

Reply:
I just split one function to two functions

Reply:
I know, but didn't the function name change to cosume_token?

Reply:
Fix later